compliance/cis_eks/rules/cis_5_4_2/data.yaml

```diff
@@ -0,0 +1,49 @@
+metadata:
+  id: c4faa477-6da2-5220-a678-0200ae5be03f
+  name: Ensure clusters are created with Private Endpoint Enabled andPublic Access Disabled (Manual)
+  profile_applicability: |
+    * Level 2
+  description: >
+    Disable access to the Kubernetes API from outside the node network if it is
+    not required.
+  rationale: >
+    In a private cluster, the master node has two endpoints, a private and
+    public endpoint. The
+    private endpoint is the internal IP address of the master, behind an internal load balancer
+    in the master's VPC network. Nodes communicate with the master using the private
+    endpoint. The public endpoint enables the Kubernetes API to be accessed from outside the
+    master's VPC network.
+    Although Kubernetes API requires an authorized token to perform sensitive actions, a
+    vulnerability could potentially expose the Kubernetes publically with unrestricted access.
+    Additionally, an attacker may be able to identify the current cluster and Kubernetes API
+    version and determine whether it is vulnerable to an attack. Unless required, disabling
+    public endpoint will help prevent such threats, and require the attacker to be on the
+    master's VPC network to perform any attack on the Kubernetes API.
+  audit: ""
+  remediation: ""
+  impact: |
+    Configure the EKS cluster endpoint to be private. See Modifying Cluster
```

Comment:
Modifying Cluster Endpoint Access should be link

Reply:
same